tests/simpleTests.R

```diff
@@ -23,47 +23,80 @@ if (Sys.getenv("Run_RPushbullet_Tests")=="yes") {
     ## Check the internal helper functions
     RPushbullet:::.getKey()
     RPushbullet:::.getDevices()
+    RPushbullet:::.getTestEmail()
+    RPushbullet:::.getTestChannel()
+
+    ## Check for missing options needed for testing
+    if(length(RPushbullet:::.getTestEmail())==0){
+        stop("Valid test e-mail needs to be specified in ~/.rpushbullet.json to run tests.")
+    }
+    if(length(RPushbullet:::.getTestChannel())==0){
+        stop("Valid test e-mail needs to be specified in ~/.rpushbullet.json to run tests.")
+    }
 
     ## Show the list of devices registered to the key
     require(RJSONIO)
     str(pbGetDevices())
 
     ## Post a note item
-    res <- fromJSON(pbPost("note", "A Simple Test",
+    result <- fromJSON(pbPost("note", "A Simple Test",
                            "We think this should work.\nWe really do.")[[1]])
-    str(res)
-    ## storing this test result to allow us to use active user's email for testing below. 
+    if(!result$active){
+        stop("Test Failed.")
+    }
+    ## storing this test result to allow us to use active user's email for testing below.
 
```

Comment:
If this first test failed for me yesterday, and if if you only test for email and channel, I suspect it will still fail for me.

Which means that we borked what was working behaviour -- fallback to default device.